package.json

```diff
@@ -18,7 +18,7 @@
     "tidy": "rm -rf modules/**/lib coverage modules/**/coverage .nyc_output modules/**/.nyc_output modules/**/*.log",
     "clean": "rm -rf node_modules modules/**/node_modules && yarn run tidy",
     "lint": "yarn workspaces run lint",
-    "test": "mocha",
+    "test": "mocha --timeout 10000",
```

Comment:
Did you try it with `yarn test-ci`?  That original writer of the code uses that in `.circle/config.yml` to run tests after a PR has been merged.

Reply:
The `yarn test-ci` use the `mocha-junit-reporter` (a link [here](https://www.npmjs.com/package/mocha-junit-reporter)) and not affect the timeout. 
I tried and it works on my local PC but on a less performing PC, the problem occurs randomly. 

For the purposes of the pipeline on GitHub, I don't see the reporter file needed. Maybe we can leave it in case it is reused with circle ci.
But to check before merging I think `yarn run test` is enough, however, the purpose fulfills it in the same way.

Reply:
Done 😉